pyproject.toml

```diff
@@ -0,0 +1,52 @@
+[tool.poetry]
+name = "c7n examples repo"
+version = "0.1.0"
+description = "CExample policies, infrastructure, and helpful automation for learning c7n"
+authors = ["Cloud Custodian Project"]
+readme = "README.md"
+homepage = "https://cloudcustodian.io"
+repository = "https://github.com/cloud-custodian/examples"
+
+[tool.poetry.dependencies]
+python = "^3.7"
+c7n = "^0.9.14"
+appdirs = "1.4.4"
```

Comment:
We can probably remove some/all of the deps listed after `c7n` here in a follow-up change. Poetry will keep track of pinned versions of lower level dependencies in its lock file so we don't have to pin as strictly or list all the individual side packages here.

Reply:
Cool. I'll file a ticket for that!